.github/workflows/ci.yaml

```diff
@@ -0,0 +1,31 @@
+name: scala-lang docs
+
+on:
+  pull_request:
+  schedule:
+    - cron: '00 0 * * *'
+
+jobs:
+  build-and-validate-docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '2.5.8'
+          bundler-cache: true
+      - name: Build documents
+        run: |
```

Comment:
I see in the Travis-CI script that we also run `bundle update --bundler`. Is this step not necessary here as well?

https://github.com/scala/scala-lang/blob/a2252985ad8708b60cbda666aa58c087370f3b83/.travis.yml#L8

Reply:
My understanding for that line is to update bundles (dependencies) while CI.
I do not know a motivation behind that.
Usually, CI uses dependencies as defined.

Reply:
Added that step.
https://github.com/scala/scala-lang/pull/1276/commits/4e7ce9b406a4c57b8b7ecbe7f29d96414e01af92
This step may be replaced with other (e.g. dependabot), but I am not sure.